File: Example4OC/Example4OC/Classes/Map/BVPreViewMapController.m
//
//  BVPreViewMapController.m
//  LibosExample
//
//  Created by 黑眸智能 on 2021/11/2.
//

#import "BVPreViewMapController.h"

@interface BVPreViewMapController ()

@end

@implementation BVPreViewMapController

- (void)viewDidLoad {
    [super viewDidLoad];
    // Do any additional setup after loading the view.
}

- (NSString *)getNavTitle {
    return @"预览地图模式";
}

- (BVSweeperMapViewMode)getMapMode {
    return BVSweeperMapViewModeMapPreviewMode;
}

@end

File: Example4Swift/Example4Swift/Classes/Map/BVPreViewMapController.swift
//
//  BVPreViewMapController.swift
//  Libos
//
//  Created by 黑眸智能 on 2021/11/2.
//

import UIKit
import BVSweeperKit

class BVPreViewMapController: BVMapController {

    override func viewDidLoad() {
        super.viewDidLoad()
    }
    
    override func getNavTitle() -> String {
        "预览地图模式"
    }
    
    override func getMapMode() -> BVSweeperMapViewMode {
        .mapPreviewMode
    }

}
